src/pluto/pluto.py

```diff
@@ -421,111 +322,319 @@ def generate_graph(v1, v2):
+                return df.groupby(['field']).size().to_frame('size').reset_index()
+            
+            def empty_message(applied, version):
+                if applied:
+                    st.info(f"No Corrections introduced in Version {version} were applied.")
+                else:
+                    st.info(f"All Corrections introduced in Version {version} were applied.")
+
+            df = filter_by_version(df, version)
+
+            if df.empty:
+                empty_message(applied, version)
+            else:
+                figure = generate_graph(field_correction_counts(df), version, applied)
+                st.plotly_chart(figure)
+
+                display_corrections_df(df)
+
+
+        st.header("Manual Corrections")
+
+        applied_corrections = data['pluto_corrections_applied']
+        not_applied_corrections = data['pluto_corrections_not_applied']
+
+        if applied_corrections is None or not_applied_corrections is None:
+            st.info("There are no available corrections reports for this branch.")
+            return
+            
+        st.markdown(
+            """
+            PLUTO is created using the best available data from a number of city agencies. To further
+            improve data quality, the Department of City Planning (DCP) applies changes to selected field
+            values.
+
+            Each Field Correction is labeled for a version of PLUTO. For programmatic changes, this is version in which the programmatic change was
+            implemented. For research and user reported changes, this is the version in which the BBL
+            change was added to PLUTO_input_research.csv.
+
+            For more information about the structure of the pluto corrections report,
+            see the [Pluto Changelog Readme](https://www1.nyc.gov/assets/planning/download/pdf/data-maps/open-data/pluto_change_file_readme.pdf?r=22v1).
+            """
+        )
+
+        version_dropdown = np.insert(np.flip(np.sort(data["pluto_corrections"].version.dropna().unique())), 0, 'All')
+        version = st.sidebar.selectbox("Filter the field corrections by the PLUTO Version in which they were first introduced", version_dropdown)
+        
+        st.subheader("Manual Corrections Applied", anchor="corrections-applied")
+        
+        create_corrections_section(applied_corrections, version, applied=True)
+
+        st.subheader("Manual Corrections Not Applied", anchor="corrections-not-applied")
+
+        st.markdown(
+            """ 
+            For each record in the PLUTO Corrections table, PLUTO attempts to correct a record by matching on the BBL and the 
+            Old Value column. As the underlying datasources change and improve, PLUTO records may no longer match the old value 
+            specified in the pluto corrections table. The graph and table below outline the records in the pluto corrections table that failed to be applied for this reason.
+            """
+        )
```

Comment:
Can these elements also go inside the `create_corrections_section`? Conceptually to me they are part of the "corrections section" as a part of the app. 
Maybe a dictionary that maps the text of "applied" and "not applied" to the correct subheader/markdown elements? Maybe that's overkill and all we need is and if/else block. Curious about your thoughts

Reply:
I think I disagree here, but its maybe because create_corrections_section is misnamed. It's a function for wrapping the logic around displaying a graph and underlying dataframe. If we wanted to encapsulate this logic in functions, i'd suggest seperate wrapper functions around create_corrections_section instead like
```
def corrections_applied_section():
  # markdown
  # display_corrections_graphs
  # info
```

Reply:
On second thought, I like the dictionary idea I think - will try to make this cleaner